### PR TITLE
[wip] Added database config and connection pool

### PR DESCRIPTION
**Pull request by lindsaymb15**
DB config and connection pool added.

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	"os"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+var DB *gorm.DB
+
+var (
+	host     = os.Getenv("HOST")
```
Comment (javalosak) [inline database/database.go:15]: ```DB_HOST```

Comment (javalosak) [reply database/database.go:15]: Agregarlos al readme

File: database/database.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+func InitDatabase() {
+	initEnvVariables()
+	psqlConnInfo := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s", DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME)
+	database, err := gorm.Open("postgres", psqlConnInfo)
+	helpers.HandleErr(err)
+
+	//change to use global variables
+	database.DB().SetMaxIdleConns(20)
```
Comment (asanchezhed) [inline database/database.go:37]: Can we set an env variable for these values?

File: README.md [hunk 3]
```diff
@@ -16,5 +16,6 @@
 
 ## Run it locally
 
+- add a .env file on the root folder with the DB config variables (DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME)
```
Comment (asanchezhed) [inline README.md:19]: can we add an env.example file with the variable names we use on the project so every time a new dev comes, they will only need to fill out the file with their credentials?

**Review by asanchezhed — CHANGES_REQUESTED** [review]

**Review by asanchezhed — APPROVED** [review]

**Pull request merged**